src/DriverConfig.php

```diff
@@ -45,13 +41,4 @@ private function getDriverOptions(bool $isMainNet): array
             ?: $this->mainnet
                 ?: [];
     }
-
-    public static function __set_state(array $state): DriverConfig
```

Comment:
seems like there is not usage for this magic method in a code anymore. feel free to correct me if I missed something